tests/lib/tools/suite/journal-state/task.yaml

```diff
@@ -0,0 +1,27 @@
+summary: smoke test for the journal-state tool
+execute: |
+    # Check help
+    "$TESTSTOOLS"/journal-state -h | MATCH "usage: journal-state start-new-journalctl-log"
+    "$TESTSTOOLS"/journal-state --help | MATCH "usage: journal-state start-new-journalctl-log"
+
+    cursor1=$("$TESTSTOOLS"/journal-state get-last-journalctl-cursor)
+    test $(journalctl --cursor "$cursor1" | grep -c "$SPREAD_JOB") -eq 0
```

Comment:
As here we are doing grep after with the spread_job it shouldn't fail, the only way it could fail is we run in parallel in the same machine/instance this test but spread does not allow that. 
I added a comment now for that line